.travis.yml

```diff
@@ -2,10 +2,8 @@ sudo: required
 language: generic
 services: docker
 env:
-  - HHVM_VERSION=4.0-latest
-  - HHVM_VERSION=4.8-latest
+  - HHVM_VERSION=4.56-latest
```

Comment:
should there be something in composer.json too?

Reply:
Ah sorry missed this before merging it. What would the change be in composer.json? I don't think we specify a minimum hhvm version in there although I probably need to bump the dependencies